tests/components/more_channels.test.jsx

```diff
@@ -7,25 +7,52 @@ import {shallow} from 'enzyme';
 import MoreChannels from 'components/more_channels/more_channels.jsx';
 import SearchableChannelList from 'components/searchable_channel_list.jsx';
 
+jest.mock('utils/browser_history', () => {
```

Comment:
Maybe we could add a reusable function to mock this since we're doing it in a few places?